presto-main/src/main/java/io/prestosql/server/PagesResponseWriter.java

```diff
@@ -41,6 +41,8 @@
 public class PagesResponseWriter
         implements MessageBodyWriter<List<SerializedPage>>
 {
+    public static final int SERIALIZED_PAGES_MAGIC = 0xfea4f001;
```

Comment:
nit: Can you use a nicer number ;) https://en.wikipedia.org/wiki/Hexspeak

Reply:
this will be "fearful", if you let yourself read it this way

Reply:
Oh yeah - looked like garbage. But now I see it. Thanks. It can stay ;)